src/rustplusEvents/message.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
             else if (message.broadcast.hasOwnProperty('teamMessage')) {
                 /* Let command handler handle the potential command */
                 CommandHandler.inGameCommandHandler(rustplus, client, message);
+                rustplus.handleGame2DiscordChat(message.broadcast.teamMessage.message)
```

Comment:
Create a seperate file for the handler in the handler folder like its done with CommandHandler.inGameCommandHandler